pkg/kubelet/kubelet.go

```diff
@@ -2878,6 +2879,55 @@ func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, v1.PodResizeStatus, string)
 	return true, v1.PodResizeStatusInProgress, ""
 }
 
+// Get pod resizing info in json format
+func getPodResizeInfo(podName string) (string, error) {
```

Comment:
As mentioned above, rather than retrieving the allocation info from the cache here, just pass the `allocatedPod` to this method, and generate the summary from that.